src/main/java/kuromi/command/Command.java

```diff
@@ -0,0 +1,37 @@
+package kuromi.command;
```

Comment:
I like how u organised your classes into sub packages to make your code more maintainable!